controllers/front/return.php

```diff
@@ -44,7 +46,62 @@ public function postProcess()
         $cartId = (int) Tools::getValue('cartId');
         $order = new Order($this->getOrderId($cartId));
 
-        if (!$order->id) {
+        /** @var SaferPayTransactionAssertion $transactionAssert */
+        $transactionAssert = $this->module->getService(SaferPayTransactionAssertion::class);
+
+        $assertResponseBody = $transactionAssert->assert($cartId);
+        $transactionStatus = $assertResponseBody->getTransaction()->getStatus();
+
+        if (Tools::getValue('isBusinessLicence')) {
```

Comment:
Might be usefull to leave note here for other developers why you are creating order if user has bussiness licence